docs/api.md

```diff
@@ -1100,6 +1114,56 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+# Group Datasets
```

Comment:
How about moving this to after `Submissions`? I think we want to keep things in the order that they are likely to matter to people.

Reply:
moved